server/__init__.py

```diff
@@ -81,14 +81,14 @@ def encode_queues(queues):
 
 
 def run_lobby_server(
-    address: (str, int),
-    database: FAFDatabase,
-    player_service: PlayerService,
-    games: GameService,
-    loop,
-    nts_client: Optional[TwilioNTS],
-    geoip_service: GeoIpService,
-    ladder_service: LadderService
+        address: (str, int),
```

Comment:
What's with the double indents? I prefer single, and for what it's worth so does `black`

Reply:
That's a good question. PyCharm formatted it that way, I think it's just conforming to PEP8, but I'm amenable to either single or double, no strong feelings one way or the other.

Reply:
Ok, let's keep it single then since that's what I've been doing in the rest of the code.